adyen.php

```diff
@@ -113,6 +147,11 @@ public function install()
     public function uninstall()
     {
         // TODO: delete adyen configurations (api-key)
+        $db = Db::getInstance();
+        /** @noinspection SqlWithoutWhere SqlResolve */
+//        $db->execute('DELETE FROM `' . _DB_PREFIX_ . 'adyen_notification`');
```

Comment:
```suggestion

```

Reply:
I should have let the command there actually. Thanks for picking this up.